webhook/webhook.go

```diff
@@ -74,7 +80,7 @@ func (dbpi *DBProxyInjector) Handle(ctx context.Context, req admission.Request)
 
 	err := dbpi.Decoder.Decode(req, pod)
 	if err != nil {
-		dbProxyLog.Info("Sdecar-Injector: cannot decode")
+		dbProxyLog.Error(err, "Sdecar-Injector: cannot decode")
```

Comment:
Good opportunity to fix the typo :)
```suggestion
		dbProxyLog.Error(err, "Sidecar-Injector: cannot decode")
```